onnx/defs/schema.h

```diff
@@ -463,13 +463,39 @@ class OpSchema {
   };
 };
 
+typedef std::unordered_map<
+    std::string,
+    std::unordered_map<std::string, std::map<OperatorSetVersion, OpSchema>>>
+    OpName_Domain_Version_Schema_Map;
+    
+
 /**
  * @brief A registry to hold all the operator schemas.
  */
 class OpSchemaRegistry {
  public:
-  // Update this when you make BC-breaking changes to the operator schema
-  constexpr static int version = 2;
+  class DomainToVersionRange {
+   public:
+    DomainToVersionRange() {
+      // Increase the highest version when you make BC-breaking changes to the
+      // operator schema on specific domain. Update the lowest version when it's
+      // determined to remove too old version history.
+      map_[""] = std::make_pair(1, 2);
```

Comment:
IIUC, the first element of this pair is not used by any code atm, is that right?

Reply:
it's not. Per our discussion, we may want to remove too old histories form old.cc one day, right? in that way, this lowest version may be changed accordingly. We don't want it? :)

Reply:
Nope, I'm just making sure I understood the modification properly. Standard of review for code that is not exercised yet should, IMO, be higher, since you haven't actually tested the design, and future authors need to know that this thing that doesn't seem to be used indeed is not used.